kong/dao/migrations/cassandra.lua

```diff
@@ -143,5 +143,38 @@ return {
     down = [[
       ALTER TABLE nodes WITH default_time_to_live = 3600;
     ]]
+  },
+  {
+    -- This is a 2 step migration; first create the extra column, using a cql statement
+    -- and following iterate over the entries to insert default values.
+    -- Step 1)
+    name = "2016-09-05-212515_retries_step_1",
+    up = [[
+      ALTER TABLE apis ADD retries int;
+    ]],
+    down = [[
+      ALTER TABLE apis DROP retries;
+    ]]
+  },
+  {
+    -- Step 2)
+    name = "2016-09-05-212515_retries_step_2",
+    up = function(_, _, dao)
+      local rows, err = dao.apis:find_all() -- fetch all rows
+      if err then
+        return err
+      end
+
+      for _, row in ipairs(rows) do
+        if not row.retries then  -- only if retries is not set already
+          local _, err = dao.apis:update(row, { id = row.id }, {full = true})
```

Comment:
Bug: this is not doing anything. It's probably missing a `row.retries = 5` before executing the `update(..)` method.